features/gebruik_in_lopende_tekst.feature

```diff
@@ -1,189 +1,651 @@
-      | vrouw    | man      | Jonkheer | Eigen                 | A.C. van der Veen                       | mevrouw Van der Veen                       |
-      | vrouw    | man      | Jonkheer | Partner na eigen      | A.C. van der Veen-van den Aedel         | mevrouw Van der Veen-van den Aedel         |
-      | vrouw    | man      | Jonkheer | Partner               | A.C. van den Aedel                      | mevrouw Van den Aedel                      |
-      | vrouw    | man      | Jonkheer | Partner voor eigen    | A.C. van den Aedel-van der Veen         | mevrouw Van den Aedel-van der Veen         |
+  De gebruikInLopendeTekst bij een persoon wordt gevuld door de provider om op deze wijze op eenduidige wijze een persoon te kunnen benoemen.
+
+  De gebruikInLopendeTekst kan worden gebruikt in bijvoorbeeld een zin in een brief als "In uw brief van 12 mei jongstleden, geeft u het overlijden van uw vader, de heer in het Veld, aan.", waarbij "de heer in het Veld" gehaald is uit attribuut gebruikInLopendeTekst.
+
+  # In onderstaande tabellen betekenen de afkortingen:
+  # GA = "mevrouw", "de heer" of de voorletters
+  # VL = voorletters
+  # VV = voorvoegselGeslachtsnaam
+  # GN = geslachtsnaam
+  # VP = voorvoegselGeslachtsnaam partner
+  # GP = geslachtsnaam partner
+  # AT = adelijke titel
+  # PK = predikaat
+  # HT = hoffelijkheidstitel of 'titre de courtoisie'
+
+Rule: gebruikInLopendeTekst voor een persoon zonder adellijke titel of predikaat wordt samengesteld afhankelijk van de geslachtsaanduiding en de waarde van aanduidingNaamgebruik, waarbij geldt dat:
+  - gebruikInLopendeTekst wordt voorafgegaan door "mevrouw", "de heer" of de voorletters, afhankelijk van de geslachtsaanduiding:
+    | geslachtsaanduiding | gebruikInLopendeTekst begint met (GA) |
+    | M                   | de heer                               |
+    | V                   | mevrouw                               |
+    | O                   | VL                                    |
+  - gebruikInLopendeTekst wordt samengesteld uit naamcomponenten van de persoon en eventuele (ex)partner op basis van de aanduidingNaamgebruik:
+    | aanduidingNaamgebruik | gebruikinlopendetekst |
+    | E                     | GA VV GN              |
+    | P                     | GA VP GP              |
+    | V                     | GA VP GP-VV GN        |
+    | N                     | GA VV GN-VP GP        |
+  - Een voorvoegsel wordt met kleine letters wordt geschreven
+  - De eerste naamcomponent in gebruikInLopendeTekst anders dan "mevrouw" of "de heer" begint met een hoofdletter
+  - Wanneer voorletters zijn opgenomen in de gebruikinlopendetekst, het voorvoegsel van de eerste naam in de gebruikinlopendetekst met alleen kleine letters wordt geschreven
+  - Wanneer een naamcomponent geen of een lege waarde heeft, wordt de overbodige spatie niet opgenomen: niet starten met een spatie, niet eindigen met een spatie, geen dubbele spatie, geen spatie na streepje
```

Comment:
mag/kan er wel een spatie voor het streepje voorkomen?

Reply:
Goed punt. Dat zou het geval kunnen zijn wanneer de naam volledig onbekend is (in bron staat dan een "." bij geslachtsnaam en voorvoegsel en voornamen zijn dan leeg, in de API worden die 3 naamvelden dan niet opgenomen).

Bijvoorbeeld wanneer de naam van de persoon (vrouw) volledig onbekend is en de partner heet "de Boer":

| naamgebruik | gebruikInLopendeTekst |
|--- |--- |
| E | mevrouw |
| P | mevrouw De Boer |
| V | mevrouw De Boer- |
| N | mevrouw -de Boer | 

Met uitzondering van naamgebruik "P" (partner) stel ik voor dat we gebruikInLopendeTekst niet opnemen. Ik kan daar een extra rule voor opnemen.

Een situatie waarbij geslachtsnaam leeg (standaardwaarde "." in de bron) is en voorvoegsel wel gevuld is, mag niet voorkomen (volgens BRP procedures), maar puur kijkend naar het informatiemodel kan het wel. Ik stel voor die situatie te negeren, want dat zou een evidente invoerfout zijn en dus zou in die situatie een samengestelde naam onjuist zijn (wie is "mevrouw Van der"?).

Ook kan het voorkomen dat de (geslachts)naam van de partner leeg/onbekend is en de naam van de partner wel gebruikt wordt.

Bijvoorbeeld bij de persoon (vrouw) met naam "de Boer" en de partner met onbekende/lege geslachtsnaam:

| naamgebruik | gebruikInLopendeTekst |
|--- |--- |
| E | mevrouw De Boer|
| P | mevrouw |
| V | mevrouw -de Boer | 
| N | mevrouw De Boer- |

Met uitzondering van naamgebruik "E" (eigen) stel ik voor dat we hier gebruikInLopendeTekst niet opnemen. Ik kan ook daarvoor een extra rule voor opnemen.

Reply:
@MelvLee ik heb deze regels toegevoegd. kan jij deze bekijken? Zijn deze in deze beschrijving
1. duidelijk
2. passend voor de (test)automatisering?

Reply:
Zal ik doen. Dit betekent dus ook dat aanhef en aanschrijfwijze hierop moet worden aangepast? Verder bedenk ik mij dat wij waarschijnlijk een scenario vergeten zijn, mensen met een naamketen.

Reply:
Nee hoor, ik pas.